params.yaml

```diff
@@ -159,10 +160,7 @@ model:
       - "loc_longitude"
       - "loc_latitude"
       - "loc_census_tract_geoid"
-      - "loc_env_flood_fema_sfha"
       - "loc_env_flood_fs_factor"
-      - "loc_env_flood_fs_risk_direction"
-      - "loc_env_airport_noise_dnl"
```

Comment:
Following the lead of the res model, I'm kicking out underperforming variables that are already largely covered by other vars.